package.json

```diff
@@ -0,0 +1,54 @@
+{
+  "name": "documents",
+  "version": "1.0.0",
+  "description": "Home Task 02",
+  "main": "index.js",
+  "scripts": {
+    "dev": "cross-env NODE_ENV=development webpack serve --mode development --open",
```

Comment:
I guess you can go with one way, because `cross-env NODE_ENV=development` and `--mode development` do the same thing